src/v2/Apps/Artwork/Components/ArtworkLightboxPlaceholder.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Image } from "@artsy/palette"
+import React from "react"
+import { Link } from "react-head"
+import { resized } from "v2/Utils/resized"
+
+interface ArtworkLightboxPlaceholderProps {
+  src: string
+  lazyLoad: boolean
+  preload: boolean
+}
+
+export const ArtworkLightboxPlaceholder: React.FC<ArtworkLightboxPlaceholderProps> = ({
+  src,
+  lazyLoad,
+  preload,
+}) => {
+  // Sized at 1px larger than the initial images
```

Comment:
Can we expand this comment a bit to explain how this relates LCP? The why isn't clear at a glance.

Reply:
Added a lengthier explanation above.